graylog2-server/src/main/java/org/graylog2/indexer/ranges/IndexRangeService.java

```diff
@@ -31,4 +31,5 @@
     void save(IndexRange indexRange);
 
     IndexRange calculateRange(String index);
+    IndexRange createForDeflector(String index);
```

Comment:
Turning out it's not as easy as it seems, as it currently relies on the choice of the value object in the service implementation. Using `MongoIndexRange` in a default implementation in the interface would smell broken to me. I will rename the method to a more generic name though.